exercises/roman-numerals/roman_numerals_test.go

```diff
@@ -11,13 +11,7 @@ func TestTestVersion(t *testing.T) {
 }
 
 func TestRomanNumerals(t *testing.T) {
-	tc := append(romanNumeralTests, []romanNumeralTest{
-		{0, "", true},
-		{-1, "", true},
-		{4000, "", true},
```

Comment:
And then favor changing the 4000 case to be 3001, which should also return an error. That would require updating the line shown below in example.go as well to change its invalid range to use 3001 instead of 4000 for the high boundary test. But it would be in line with the README and yet preserve the usefulness of the error return value.
> if input <= 0 || input >= 4000 {

Reply:
Agreed.

Reply:
This is all spot on, incorporated.